web/server/codechecker_server/api/report_server.py

```diff
@@ -2534,20 +2557,44 @@ def getLinesInSourceFileContents(self, lines_in_files_requested, encoding):
         self.__require_view()
         with DBSession(self._Session) as session:
             res = defaultdict(lambda: defaultdict(str))
-            for lines_in_file in lines_in_files_requested:
-                if lines_in_file.fileId is None:
-                    LOG.warning("File content requested without a fileId.")
-                    LOG.warning(lines_in_file)
-                    continue
-                sourcefile = session.query(File).get(lines_in_file.fileId)
-                cont = session.query(FileContent).get(sourcefile.content_hash)
+
+            contents = []
+            for chunk in util.chunks(
```

Comment:
This loop could be merged with the other one in line 2585 and that way only `contents_to_file_id` would be needed and `contents` could be eliminated. It would half the memory usage.
By the way shouldn't it be `file_id_to_contents`?

Reply:
True, merged it in with the chunk loop/

Reply:
Also changed the variable name, I always struggle with these, thanks.